Makefile

```diff
@@ -100,9 +104,9 @@ $(SYMTABOBJ): %$(OBJEXT): %.c
 
 $(BIN): $(SYMTABOBJ)
 ifeq ($(CONFIG_CYGWIN_WINTOOL),y)
-	$(call ARLOCK, "${shell cygpath -w $(BIN)}", $^)
+	$(call ARCHIVE, "${shell cygpath -w $(BIN)}", $^)
```

Comment:
Should we use ARCHIVE_ADD here?

Reply:
I understand this is the only call to the archive being created (it is either the previous recursive call into subdirectories or this one, according to the `if`). So in this case we need to create the library from scratch. But I'm not really familiar with this SYMTABOBJ rule.

Reply:
You were right, this needed ARCHIVE_ADD and remove the library earlier. I just pushed a change which consideres both the CONFIG_BUILD_LOADABLE and !CONFIG_BUILD_LOADABLE case.